graf2d/quartz/CMakeLists.txt

```diff
@@ -22,4 +22,5 @@ ROOT_LINKER_LIBRARY(GQuartz
     "-framework Cocoa"
   DEPENDENCIES
     Core
+    MathCore
```

Comment:
Could you split the missing dependencies in a separate PR. This may make the review of this PR easier.